File: PYTHON/1010 - Cálculo Simples.py
n1, n2, n3 = input().split(" ")
n4, n5, n6 = input().split(" ")

n1=int(n1)
n2=int(n2)
n3=float(n3)
n4=int(n4)
n5=int(n5)
n6=float(n6)

sum1 = n2 * n3
sum2 = n5 * n6  
total = sum1 + sum2

print(f"VALOR A PAGAR: R$ {total:.2f}")

File: RUBY/1010 - Cálculo Simples.rb
vet = gets.chomp.split
vet1 = gets.chomp.split

x = vet[0].strip.to_i
y = vet[1].strip.to_i
z = vet[2].strip.to_f
x1 = vet1[0].strip.to_i
y1 = vet1[1].strip.to_i
z1 = vet1[2].strip.to_f

sum = (y * z) + (y1 * z1)

puts "VALOR A PAGAR: R$ #{'%.2f' % sum}"